src/IROperator.h

```diff
@@ -820,7 +885,12 @@ inline Expr pow(Expr x, Expr y) {
     if (x.type() == Float(64)) {
         y = cast<double>(y);
         return Internal::Call::make(Float(64), "pow_f64", {x, y}, Internal::Call::Extern);
-    } else {
+    }
+    else if (x.type() == Float(16)) {
+         y = cast<float16_t>(y);
+        return Internal::Call::make(Float(16), "pow_f16", {x, y}, Internal::Call::Extern);
```

Comment:
You'll probably want to test your float16-using algorithm on an x86 CPU at some stage. On x86 most likely we'll just do the math as float32 then truncate on store, and this is what AVX2 supports and seems to encourage. You won't get bit-accurate results, but I think this falls within the bounds of our -ffast-math style behavior.